### PR TITLE
Improved ci

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
- add 4 5 and travis node version
- setup coverage
- add badges to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 PEG.js
 ======
 
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+[![Coverage Status][coveralls-badge]][coveralls-status]
```
Comment (josephfrazier) [inline README.md]: I don't think the `[coveralls-status]` bit renders properly (see [here](https://github.com/AdrieanKhisbe/pegjs/blob/0536e2c/README.md)). It's supposed to be `[coveralls-url]` instead?

**Comment by AdrieanKhisbe** [comment]
@josephfrazier, indeed, it was a stupid typo from me

**Comment by dmajda** [comment]
@AdrieanKhisbe Sorry I got to this PR so late. Quick summary:

  * I already [added new Node.js versions to `.travis.yml`](https://github.com/pegjs/pegjs/commit/6a3ede059daadb9815bd42cb6898d33a5fea40d0) independently.

  * I don’t want to have coverage measurement set up for PEG.js. Some parts of the code are not covered by tests on purpose (because the tests simply wouldn’t bring enough value), but seeing them as uncovered in a coverage report would undoubtedly drive people to try to cover them, wasting everybody’s time (they would write PRs which I would then reject).

  * I [added badges to `README.md`](https://github.com/pegjs/pegjs/commit/d56b43bb541f0cbecc94609baf220830fa263d71) now, although somewhat different set than yours. I didn’t like badges in the past (thinking they mostly clutter the GitHub page) but later in various projects I found value in having e.g. a link to project’s npm package or license information handy, so I added a few.

Given all this I’m closing this PR. Thanks for your contribution!

**Comment by AdrieanKhisbe** [comment]
@dmajda, no problem :)

I totally understand for coverage, I'm not a 100% extremist, just using to have some insight on test
(some inbetween option could to have coverage for you, but don't point them from the readme)

**Pull request closed without merging**